x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/__mocks__/es_results.ts

```diff
@@ -30,15 +31,25 @@ export const sampleSignalAlertParams = (
   filters: undefined,
   savedId: undefined,
   meta: undefined,
-  size: 1000,
 });
 
 export const sampleDocNoSortId: SignalSourceHit = {
   _index: 'myFakeSignalIndex',
   _type: 'doc',
   _score: 100,
   _version: 1,
-  _id: 'someFakeId',
+  _id: uuid.v4(),
+  _source: {
+    someKey: 'someValue',
+    '@timestamp': 'someTimeStamp',
+  },
+};
+
+export const sampleDocNoSortIdNoVersion: SignalSourceHit = {
+  _index: 'myFakeSignalIndex',
+  _type: 'doc',
+  _score: 100,
+  _id: uuid.v4(),
```

Comment:
For unit tests you usually want to avoid any type of non-repeatable data such as uuid or date times or auto generated test data. Any way you can just hard code a GUID here instead?